readme.md

```diff
@@ -373,6 +375,24 @@ body, .foo {
 ]
 ```
 
+### Compound strings
+
+`[STRING_START, <quote>], [VALUE|VALUE_REF, <argument>], [STRING_END]`
+
+When a string in CSS contains an interpolation, e.g. `"hello, ${name}"`, the string should be split up into its values and
+value refs.
+
+```js
+[
+  [STRING_START, '"'],
```

Comment:
wait, I think ISTF doesn't need to define the quote char, it can be defined in the transformer which will also know what to escape then